prog/test/vm_group.rb

```diff
@@ -70,14 +70,28 @@ def self.assemble(boot_images:, storage_encrypted: true, test_reboot: true, test
   end
 
   label def verify_host_capacity
-    hop_verify_vm_host_slices if !frame["verify_host_capacity"]
+    hop_verify_storage_backends if !frame["verify_host_capacity"]
 
     vm_cores = vm_host.vms.sum(&:cores)
     slice_cores = vm_host.slices.sum(&:cores)
     spdk_cores = vm_host.cpus.count { it.spdk } * vm_host.total_cores / vm_host.total_cpus
 
     fail_test "Host used cores does not match the allocated VMs cores (vm_cores=#{vm_cores}, slice_cores=#{slice_cores}, spdk_cores=#{spdk_cores}, used_cores=#{vm_host.used_cores})" if vm_cores + slice_cores + spdk_cores != vm_host.used_cores
 
+    hop_verify_storage_backends
+  end
+
+  label def verify_storage_backends
+    fail_test "Expected a vhost block backend" if vm_host.vhost_block_backends.empty?
+
+    Clog.emit("vm_host.vhost_block_backends not empty, verifying that SPDK is not used")
```

Comment:
```suggestion
    Clog.emit("vm_host.vhost_block_backends not empty; performing verification that SPDK is not used")
```

The previous wording could be interpreted as: the fact that vm_host.vhost_block_backends not empty is by itself verification that SPDK is not used.